benchmarks/random_allocations/random_allocations.cpp

```diff
@@ -69,7 +69,7 @@ void random_allocation_free(rmm::mr::device_memory_resource& mr,
                             SizeDistribution size_distribution,
                             size_t num_allocations,
                             size_t max_usage,  // in MiB
-                            cudaStream_t stream = 0)
+                            stream_t stream = rmm::stream_t{})
```

Comment:
Why is it `stream_t` on the left and `rmm::stream_t` on the right?

Reply:
Oops, didn't have `BUILD_BENCHMARKS` enabled in CMake